sanic_ext/extensions/openapi/types.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, **kwargs):
     def fields(self):
         return self._fields
 
-    def guard(self, fields):
+    def guard(self, fields: dict):
```

Comment:
`dict` is not an acceptable annotation.

Reply:
Removed. May I know why? And what would be a suitable type hint for it?

Reply:
1. Python 3.8 should have `Dict` and not `dict`
2. It should explain `what` it is: `Dict[str, Any]` being a broad generalization